components/properties/Cargo.toml

```diff
@@ -38,10 +38,12 @@ icu_provider = { version = "0.5", path = "../../provider/core", features = ["mac
 icu_uniset = { version = "0.4.1", path = "../../utils/uniset"}
 serde = { version = "1.0", default-features = false, features = ["derive", "alloc"], optional = true }
 zerovec = { version = "0.6", path = "../../utils/zerovec", features = ["derive"] }
+unicode-bidi = { version = "0.3.8", optional = true , default-features = false }
 
 [dev-dependencies]
 icu = { path = "../icu", default-features = false }
 icu_testdata = { version = "0.5", path = "../../provider/testdata" }
+icu_provider_fs = {version = "0.5.0", path = "../../provider/fs" }
```

Comment:
Nit: I do not think you are using this dependency, so you should remove it

Reply:
done.